### ISSUE TITLE
How to obtain the data set in experimental format

### ISSUE THREAD

**Issue by buthi** [yueliu1999/DCRN#7]
Hello, I am also doing research on graph network now. Your article is very enlightening. I would like to ask, when I was looking for data sets, I found that data sets like CITEand DBLP are all in text or json format, and I could not find dataset with adjacency matrix and attribute data. I want to know whether your experimental data is processed by yourself and then experimented with these original data sets, or where you can find the data in graph format?

**Comment by yueliu1999** [comment]
Thanks. The other datasets can be found at this link: https://github.com/yueliu1999/Awesome-Deep-Graph-Clustering#benchmark-datasets

**Comment by buthi** [comment]
> 谢谢。其他数据集可在此链接中找到：[https ://github.com/yueliu1999/Awesome-Deep-Graph-Clustering#benchmark-datasets](https://github.com/yueliu1999/Awesome-Deep-Graph-Clustering#benchmark-datasets)
Thank you for your advice! I have already star this library of yours before. I would like to know whether these datasets are processed by yourself or just existing in official datasets

**Comment by yueliu1999** [comment]
Yes, these datasets are processed by myself. I just uniformly store the official data with the npy format. The reason is that the different formats of datasets are hard to process during coding.

**Comment by buthi** [comment]
> 是的，这些数据集是我自己处理的。我只是统一用npy格式存储官方数据。原因是在编码过程中难以处理不同格式的数据集。
Thank you. May I ask how you handled the data? In addition, whether to do graph model research, the data set is their own processing into the form of graph

**Comment by yueliu1999** [comment]
I just read the original data and transfer them to numpy.ndarray,and then save them with npy format. Some data is the nature graph and others is the processed graph.
